tests/fixtures/gtfs2ntfs/minimal/output/trips.txt

```diff
@@ -1,5 +1,5 @@
 trip_id,route_id,physical_mode_id,dataset_id,service_id,trip_headsign,trip_short_name,block_id,company_id,trip_property_id,geometry_id,journey_pattern_id
-trip:3,route:2,Metro,default_dataset,service:1,pouet,,,1,,,
 trip:4,route:2,Metro,default_dataset,service:1,pouet,,,1,,,
 trip:5,route:3,Metro,default_dataset,service:2,pouet,,,2,,,
+trip:3,route:2,Metro,default_dataset,service:1,pouet,,,1,,,
```

Comment:
Isn't the sorting supposed to be stable?

Reply:
There is no sorting